Markdown link cleanup (#359)
exercism/cairo: exercises/concept/health-statistics/.docs/hints.md

- The `new()` method receives the arguments we want to instantiate a `User` instance with.
  It should return an instance of `User` with the specified name, age, and weight.

> - See [structs documentation][structs] for additional examples on defining and instantiating structs.

Reviewer: ```suggestion
- See the [structs documentation][structs] for examples defining and instantiating structs.
```

}
```

> - See [methods documentation][methods] for some more examples of defining methods on structs.

Reviewer: ```suggestion
- See the [methods documentation][methods] for more examples of defining methods on structs.
```
